libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -2802,7 +2802,9 @@ function getOutputsByType(allOutputs: OutputInfo[], type = constants.SWAGGER.TYP
     return allOutputs;
   }
 
-  return allOutputs.filter((output) => equals(type, output.type));
+  return allOutputs.filter((output) => {
+    return Array.isArray(output.type) ? false : equals(type, output.type);
```

Comment:
nit: Simplify the expression as !Array.isArray(output.type) && equals(type, output.type)